test/python/circuit/test_control_flow_builders.py

```diff
@@ -34,7 +34,7 @@
 from qiskit.test._canonical import canonicalize_control_flow
 
 
-class SentinelException(Exception):
+class SentinelError(Exception):
     """An exception that we know was raised deliberately."""
```

Comment:
I'd argue that this is a signal and not an error as well.

Reply:
Yeah. This has nothing to do with why all of these are supposed to terminate with the string `Error`. It's been reverted anyway.